### DISCUSSION TITLE
How to pause an ink story temporarily

### DISCUSSION THREAD

**Discussion by Gamma-Vector** [ephread/inkgd#64] [Q&A]
Basically, I'm trying to get to a branch point/choice in my Ink story, pause for a second, do some logic in GD Script, and then use the result of that logic to choose a branch to continue down in the Ink story.

The more detailed explanation is that I want to be able to display some choices as buttons, but have other choice branches connected to typed input from the user. Specifically, I want to be able to ask the user to input a password. If they get it right, the story continues down path A, if they get it wrong, path B.

But I can't figure out how to pause the Ink Player while I pop up an input box and prompt the user to enter something. The program just hangs.

Is this a simple thing that I've simply missed how to do, or is this not really possible?

EDIT: I've considered just trying to use multiple Ink stories (End story A, then do the password, then start story B or C depending on the result) but...I actually need to do this password thing a whole bunch of times, and it feels like multiple stories would wind up being a very clunky way to go about it.

EDIT 2: Please disregard. I don't know what was causing it, but pasting the exact same script into a fresh project has fixed the problem. I don't think it was actually on inkgd's end.

**Comment by francoisdlt** [comment]
i don't understand what you mean by "pause" and "hang", because the ink player only reacts when you pull something from it or push a choice into it...

**Reply by Gamma-Vector** [reply]
Turns out the problem was somewhere else in my code. Sorry for the trouble.
